tests/pyreverse/functional/class_diagrams/annotations/method_annotation.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+# pylint: disable-next=import-error
+from lib import Banana, Coconut, Sound  # type: ignore[import]
+
+
+class Monkey:
+    def eat(self, food: Banana | Coconut) -> None:
```

Comment:
Seems like you're handling ``|`` it made me think of this stackoverflow question : https://stackoverflow.com/questions/72388388/. Maybe we can add some test case with Union too ?

Reply:
Adding a corresponding test case is a good idea!

The alternative union syntax however gives me some headaches for the tests, as the output differs for different Python versions. I will have to take a closer look.